Extensions/xkit_patches.js

```diff
@@ -708,26 +715,45 @@ XKit.tools.dump_config = function(){
 				},
 
 				/**
-				 * Sets the content of the post window.
+				 * Uses sneaky HTML editor magic to handle singlequote issues with editable reblogs
 				 * @param {String} new_content
 				 */
-				set_content_html_sneak: function(new_content) {
-					var html_or_markdown = 'html';
-					XKit.tools.add_function(function(){
-						var new_content = add_tag[0];
-						var html_or_markdown = add_tag[1];
-						var editor_div = document.getElementsByClassName("ace_editor");
-						if (html_or_markdown === "Markdown") {
-							new_content = require('to-markdown').toMarkdown(new_content);
-						}
-						if (editor_div.length === 1) {
-							var editor = window.ace.edit(editor_div[0]);
-							editor.setValue(new_content);
-							setTimeout(function(){
-								jQuery(".ace_marker-layer").empty();
-							}, 500);
+				set_content_html_sneak: function() {
```

Comment:
name is un-descriptive—it does a lot of stuff that doesn't have to do with "setting the content html", and in fact never sets the content html at all.